testdata/prometheus/prometheus.yml

```diff
@@ -34,3 +34,235 @@ scrape_configs:
+  static_configs:
+  - targets:
+    - 127.0.0.1:32772
+    labels:
+      agent_id: /agent_id/8a3b1334-17b1-44d1-9208-1d557f933fc6
+      agent_type: mysqld_exporter
+      instance: /agent_id/8a3b1334-17b1-44d1-9208-1d557f933fc6
+      key1: value1
+      key2: value2
+      node_id: /node_id/3871629a-e57c-45e7-a4f5-d0441e667826
+      node_name: remote node
+      node_type: container
+      service_id: /service_id/3bd1b3b7-265a-40d9-943e-c8c3d25dcde5
+      service_name: 192.168.0.1
+      service_type: mysql
+- job_name: mysqld_exporter_agent_id_ee181120-3f7b-4221-a640-ecfbf6f585e6_hr
+  scrape_interval: 1s
+  scrape_timeout: 1s
+  metrics_path: /metrics-hr
+  static_configs:
+  - targets:
+    - 127.0.0.1:32771
+    labels:
+      agent_id: /agent_id/ee181120-3f7b-4221-a640-ecfbf6f585e6
+      agent_type: mysqld_exporter
+      instance: /agent_id/ee181120-3f7b-4221-a640-ecfbf6f585e6
+      node_id: /node_id/8554ad02-62e5-4328-a82c-cc375e70d52e
+      node_name: remote node 2
+      node_type: container
+      service_id: /service_id/5b135171-5334-413b-aba9-a2831bfda6a9
+      service_name: third-mysql
+      service_type: mysql
+- job_name: mysqld_exporter_agent_id_ee181120-3f7b-4221-a640-ecfbf6f585e6_mr
+  scrape_interval: 5s
+  scrape_timeout: 4s
+  metrics_path: /metrics-mr
+  static_configs:
+  - targets:
+    - 127.0.0.1:32771
+    labels:
+      agent_id: /agent_id/ee181120-3f7b-4221-a640-ecfbf6f585e6
+      agent_type: mysqld_exporter
+      instance: /agent_id/ee181120-3f7b-4221-a640-ecfbf6f585e6
+      node_id: /node_id/8554ad02-62e5-4328-a82c-cc375e70d52e
+      node_name: remote node 2
+      node_type: container
+      service_id: /service_id/5b135171-5334-413b-aba9-a2831bfda6a9
+      service_name: third-mysql
+      service_type: mysql
+- job_name: mysqld_exporter_agent_id_ee181120-3f7b-4221-a640-ecfbf6f585e6_lr
+  scrape_interval: 1m
+  scrape_timeout: 10s
+  metrics_path: /metrics-lr
+  static_configs:
+  - targets:
+    - 127.0.0.1:32771
+    labels:
+      agent_id: /agent_id/ee181120-3f7b-4221-a640-ecfbf6f585e6
+      agent_type: mysqld_exporter
+      instance: /agent_id/ee181120-3f7b-4221-a640-ecfbf6f585e6
+      node_id: /node_id/8554ad02-62e5-4328-a82c-cc375e70d52e
+      node_name: remote node 2
+      node_type: container
+      service_id: /service_id/5b135171-5334-413b-aba9-a2831bfda6a9
+      service_name: third-mysql
+      service_type: mysql
+- job_name: node_exporter_agent_id_ecbc388c-265e-44a5-9c2b-e12947b3fa74
+  scrape_interval: 1s
+  scrape_timeout: 1s
+  metrics_path: /metrics
+  static_configs:
+  - targets:
+    - 192.168.1.22:32770
+    labels:
+      agent_id: /agent_id/ecbc388c-265e-44a5-9c2b-e12947b3fa74
+      agent_type: node_exporter
+      instance: /agent_id/ecbc388c-265e-44a5-9c2b-e12947b3fa74
+      machine_id: |
+        /machine_id/619d68d22b96945324eb45735bad3294
+      node_id: /node_id/372b101e-9ef4-4d70-874c-e24329dc3eaa
+      node_name: nurlan-XPS-15-9570
+      node_type: generic
```

Comment:
I think that file should not be updated in that PR. Especially given a `node_name` above. :)

Reply:
yeah, that's my fault. Could we add this file to gitignore? and generate from example file?

Reply:
Currently, that file is used and sometimes changed – when we change Prometheus generation code. We can't add it to gitignore without changing how Prometheus tests work. In other words, not now.

Reply:
I mean create new file `prometheus.example.yml`, remove `prometheus.yml` from repo and copy it to `prometheus.yml` before tests and before run pmm-managed.

Reply:
That might work